plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/AmazonQLanguageClient.kt

```diff
@@ -45,4 +52,19 @@ interface AmazonQLanguageClient : LanguageClient {
 
     @JsonNotification(CHAT_SEND_CONTEXT_COMMANDS)
     fun sendContextCommands(params: LSPAny): CompletableFuture<Unit>
+
+    @JsonNotification(DID_COPY_FILE)
+    fun copyFile(params: CopyFileParams): CompletableFuture<Unit>
```

Comment:
```suggestion
    fun copyFile(params: CopyFileParams)
```

notification handlers actually do not need to return a future